account-kit/react/src/components/auth/card/error/connection-error.tsx

```diff
@@ -74,15 +100,36 @@ export const ConnectionError = ({
       <h2 className="font-semibold text-lg text-center">{getHeadingText}</h2>
       <p className="text-sm text-center text-fg-secondary">{getBodyText}</p>
       <Button className="mt-3" onClick={handleTryAgain}>
-        {ls.error.cta.tryAgain}
+        {customErrorMessage?.tryAgainCTA ?? ls.error.cta.tryAgain}
       </Button>
       <Button
         onClick={handleUseAnotherMethod}
         variant={"social"}
         className="border-0 bg-btn-secondary"
       >
-        {ls.error.cta.useAnotherMethod}
+        {customErrorMessage?.useAnotherMethodCTA ??
+          ls.error.cta.useAnotherMethod}
       </Button>
     </div>
   );
 };
+
+const clearWalletConnectStore = () => {
+  // Open Wallet Connect Indexed DB
+  const dbOpenRequest = indexedDB.open("WALLET_CONNECT_V2_INDEXED_DB");
+
+  dbOpenRequest.onsuccess = () => {
+    const db = dbOpenRequest.result;
+
+    const txn = db.transaction(["keyvaluestorage"], "readwrite");
+
+    const store = txn.objectStore("keyvaluestorage");
+
+    // Clear Store
+    store.clear();
+  };
+
+  dbOpenRequest.onerror = () => {
+    console.error("Error opening Wallet Connect DB. Cannot clear store.");
+  };
+};
```

Comment:
this feels like something that should live in the `import { disconnect } from "@account-kit/core";` function instead